### PR TITLE
pushed RecordViewModel

### PR DESCRIPTION
**Pull request by fvkxvsop**
Developed RecordViewModel which renders and displays relevant information regarding objects inhereting from Record class.

### CODE REVIEW

**Review by dqieu — APPROVED** [review]
awaiting backend/abr implementation to test the viewmodel but lgtm

**Review by varygx — APPROVED** [review]
Viewmodel looks good, has basic documentation

File: src/main/java/interface_adaptors/RecordViewModel.java [hunk 1]
```diff
@@ -1,4 +1,99 @@
 package interface_adaptors;
 
-public class RecordViewModel {
+import javax.imageio.ImageIO;
+import javax.swing.*;
+import java.awt.*;
+import java.util.ArrayList;
+
+import entities.Record;
```
Comment (Starliyiyu) [inline src/main/java/interface_adaptors/RecordViewModel.java:8]: entities shouldn't be in imported/ manipulated with in the interface_adapters layer

**Review by wilso16 — DISMISSED** [review]
- Update file so that the entities are not imported (use the attributes of the Record class instead)
- Is RecordViewModel supposed to be singleton?
- I know you are working on fixing these updates right now so approving for testing purposes

**Review by dqieu — APPROVED** [review]
awaiting backend/abr implementation to test the viewmodel but lgtm

**Review by varygx — APPROVED** [review]
Went in and fixed merge conflict for file.

**Review by cynthiaxsa — APPROVED** [review]
pog

**Review by wilso16 — APPROVED** [review]
- Reorganization of branches, nothing appears to be changed since last approve

**Pull request merged**